core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -76,6 +76,7 @@
 import lombok.experimental.UtilityClass;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.tuple.Pair;
+import org.apache.commons.math3.analysis.function.Exp;
```

Comment:
a mistake?

Reply:
Removed in https://github.com/opensearch-project/sql/pull/1467/commits/4eefd7dab77306ce99ac4ea477736b7bb597175b